src/servers/graphql-server.ts

```diff
@@ -86,16 +94,23 @@ const setGqlContext = async (
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
-  const tokenPayload = req.token
-
-  const body = req.body ?? null
-
   const ipString = isDev
     ? req.ip
     : req.headers["x-real-ip"] || req.headers["x-forwarded-for"]
-
   const ip = parseIps(ipString)
-
+  const body = req.body ?? null
+  let tokenPayload
+  const kratosCookie = req.cookies.ory_kratos_session
```

Comment:
after the cookie is set, and the client make request to `/graphql`, oathkeeper should parse the cookie and return the JWT. so there should be no change in the way request are been handled.